CalendarAppTests/SampleEvents.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SampleEvents.swift
+//  CalendarAppTests
+//
+//  Created by Angelina Zhang on 7/20/22.
+//
+
+import Foundation
+@testable import CalendarApp
+
+enum SampleEvents {
+    static let valid = Event()
+    private static let oneHour = TimeInterval(3600)
+    private static let oneDay = TimeInterval(86400)
+    
+    static func makeEvent(on date: Date) -> Event {
+        let event = Event()
+        event.startDate = date
+        event.endDate = date.advanced(by: oneHour)
+        return event
+    }
+    
+    static func makeEvent(on date: Date,
+                          withDays days: Double,
+                          withHours hours: Double) -> Event {
```

Comment:
I might change this api a bit. 
```
makeEvent(
    start: Date,
    duration: TimeInterval
) {
    let event = Event()
    event.startDate = date
    event.endDate = date.advanced(by: duration)
    return event
}

// if you want the convenience it can delegate to the other.
makeEvent(
    start: Date,
    days: Int,
    hours: Int
) {
    makeEvent(
        start: start,
        duration: (days * 24 + hours) * 60
    )
}
```